packages/flutter/test/rendering/sliver_fixed_extent_layout_test.dart

```diff
@@ -103,3 +171,17 @@ class TestRenderSliverBoxChildManager extends RenderSliverBoxChildManager {
   @override
   void setDidUnderflow(bool value) { }
 }
+
+class TestRenderSliverFixedExtentBoxAdaptor extends RenderSliverFixedExtentBoxAdaptor {
+  TestRenderSliverFixedExtentBoxAdaptor() 
```

Comment:
I think the analyzer is complaining about 

```suggestion
  TestRenderSliverFixedExtentBoxAdaptor()
```

Reply:
cf https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket.appspot.com/8865114836733794336/+/steps/prepare_environment/0/steps/Analyze/0/steps/analyze/0/stdout

Reply:
Thanks, it's fixed.